scripts/verify-swaggo-fmt.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# This script checks for swaggo and then uses the found swaggo to run swaggo fmt, failing if the command makes any changes to the files.
+
+swaggo="swag"
+
+# Test if swaggo is installed as a binary or a go module.
+if command -v swag &> /dev/null; then
+    # If the user has installed swaggo as a binary, use the binary.
+    swaggo=$(command -v swag)
+elif [ -n "$GOPATH" ] && [ -f "$GOPATH/bin/swag" ]; then
+    swaggo="$GOPATH/bin/swag"
+elif [ -n "$GOBIN" ] && [ -f "$GOBIN/swag" ]; then
+    swaggo="$GOBIN/swag"
+fi
+
+# If swaggo is not installed, exit with an error.
+if ! command -v "$swaggo" &> /dev/null; then
+    echo "swaggo is not installed."
+    exit 1
+fi
+
+# Compare the diff before and after running swaggo fmt. We only care about if the modified files has changed.
+before=$(git diff --diff-filter=M)
+$swaggo fmt -g service/service.go
```

Comment:
It expands the variable to use it as a command. If I had just put "swaggo"- it would just try running a binary named swaggo if it could find one.

I think alternatively to storing the variable and using the command like this- people define it as aliases or something.